runtime/executor/src/main/java/edu/snu/onyx/runtime/executor/data/DataUtil.java

```diff
@@ -240,20 +240,18 @@ public T next() {
    * Creates new OutputStream which writes compressed data onto the stream.
    *
    * @param out the original {@link OutputStream}.
-   * @param compressor compress strategy for compressing stream data.
+   * @param compression compress strategy for compressing stream data.
    * @return {@link OutputStream} which writes compressed data.
    * @throws IOException if fail to compress data.
    */
-  OutputStream createOutputStream(final OutputStream out, final Compressor compressor)
+  OutputStream createOutputStream(final OutputStream out, final Compression compression)
```

Comment:
Why do we still need these methods? According to our [design doc](https://docs.google.com/document/d/1uhQVbAR6OIxLj1XHdcrMsT40N4djqOU2x_yhmYKpm0A/edit), it seems that you are planning to implement and use a new interface `Filter`, instead of these methods.